src/views/Vote/VoteForm/VoteForm.vue

```diff
@@ -36,19 +36,14 @@ const Votings = namespace("votings");
   },
 })
 export default class VoteForm extends Vue {
-  private input = {
-    score: 0,
-    comment: "",
-    anonymous: false,
-    votingId: "",
-    groupId: "",
-  } as CreateVoteInput;
-
   private showPreview = false;
 
   @Prop(Object)
   private readonly group!: Group;
 
+  @Prop(Number)
+  private readonly voteScore: number = 3;
```

Comment:
This shouldn't be here, Vue is complaining (Check browser console). You are mutating the value. Whether you set it in the `pug` file `voteScore || 3` or we set it in the router, or we make sure links to vote come with a vote value.

Reply:
![console](https://user-images.githubusercontent.com/1414131/84624990-aefed680-aee2-11ea-9c60-185d7470d353.png)